fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -128,17 +128,20 @@ module Generators =
               (fun i (v : RT.DType) -> (id i, $"{v.toOldString().ToLower()}_{i}"))
               paramTypes
 
-          // FSTODO: occasionally use the wrong return type
-          // FSTODO: can we use the argument to get this type?
+          let! returnType =
+            Gen.frequency [ (98, Gen.constant returnType)
+                            (2, G.RuntimeTypes.dType) ]
+
+          // FSTODO: can we use the argument to get this type? WHATISTHIS
           let! body = genExpr' returnType size
           return RT.ELambda(gid (), parameters, body)
         | RT.TBytes ->
-          // FSTODO: this doesn't really do anything useful
+          // FSTODO: this doesn't really do anything useful WHATISTHIS
           let! bytes = Arb.generate<byte []>
           let v = RT.EString(gid (), Base64.defaultEncodeToString bytes)
           return callFn "String" "toBytes" 0 [ v ]
```

Comment:
I'm not sure what the expectation is here.
I recall there being an issue with bytes interoping with OCaml backend well, which is why there's a `containsBytes` filter elsewhere. Was the intention behind this to get around that?
In any case, what should be done?

Reply:
This is here because there isn't a literal byte representation in Dark. I don't remember why that comment is there, but the intention is to expand our byte level testing to include non-stringable bytes (to test edges of conversions to strings, and just be more robust in general). If we're able to remove `containsBytes` or reduce its scope, all the better.